src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number | string;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape: string = 'triangle',
```

Comment:
[CODE KNOWLEDGE] - TypeScript implicitly creates a property on the class when you declare a constructor parameter with an access modifier. You don't need to assign 'shape' explicitly if you declare it as 'public' in the constructor.